packages/manager/apps/container/src/cookie-policy/CookiePolicy.spec.tsx

```diff
@@ -0,0 +1,104 @@
+      <CookiePolicy onValidate={onValidate} shell={shell} />
+    </ApplicationContext.Provider>
+  );
+};
+
+const mockedShell = (region: string) => ({
+  getPlugin: (plugin: string) => ({
+    environment: {
+      getEnvironment: () => ({
+        user: {
+          ovhSubsidiary: region,
+        } as User,
+        getRegion: () => region,
+      } as Environment),
+    },
+    tracking: {
+      setRegion: vi.fn(),
+      init: trackingInit,
+      setEnabled: trackingSetEnabled,
+      onConsentModalDisplay: trackingOnConsentModalDisplay,
+    },
+  }[plugin]),
+});
+
+vi.mock('@ovh-ux/shell', () => ({
+  initShell: vi.fn()
+}));
+
+vi.mock('react-cookie', () => ({
+  useCookies: vi.fn(),
+}));
+
+
+describe('CookiePolicy.component', () => {
+
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  it('should init tracking if cookie is valid and region is not US', async () => {
+
+    (await import('react-cookie')).useCookies = vi.fn().mockReturnValue([{ MANAGER_TRACKING: '1', }, vi.fn()]);
+    (await import('@ovh-ux/shell')).initShell = vi.fn().mockResolvedValue(mockedShell('EU'));
+    void renderCookiePolicy();
+    await waitFor(() => {
+      expect(trackingInit).toHaveBeenCalled();
+      expect(trackingSetEnabled).not.toHaveBeenCalled();
+      expect(trackingOnConsentModalDisplay).not.toHaveBeenCalled();
+    }, { timeout: 2000 });
+  });
+
+  it('should init tracking if cookie is invalid and region is US', async () => {
+
+    (await import('react-cookie')).useCookies = vi.fn().mockReturnValue([{ MANAGER_TRACKING: '0', }, vi.fn()]);
+    (await import('@ovh-ux/shell')).initShell = vi.fn().mockResolvedValue(mockedShell('US'));
+    void renderCookiePolicy();
+    await waitFor(() => {
+      expect(trackingInit).toHaveBeenCalled();
+      expect(trackingSetEnabled).not.toHaveBeenCalled();
+      expect(trackingOnConsentModalDisplay).not.toHaveBeenCalled();
+    }, { timeout: 2000 });
+  });
+
+  it('should show consent modal if cookie is null and region is not US', async () => {
+
+    (await import('react-cookie')).useCookies = vi.fn().mockReturnValue([{ MANAGER_TRACKING: null, }, vi.fn()]);
+    (await import('@ovh-ux/shell')).initShell = vi.fn().mockResolvedValue(mockedShell('EU'));
+    void renderCookiePolicy();
+    await waitFor(() => {
+      expect(trackingInit).not.toHaveBeenCalled();
+      expect(trackingSetEnabled).not.toHaveBeenCalled();
+      expect(trackingOnConsentModalDisplay).toHaveBeenCalled();
+    }, { timeout: 2000 });
+  });
+
+  it('should disable tracking plugin if cookie is not valid or null and region is not US', async () => {
+
+    (await import('react-cookie')).useCookies = vi.fn().mockReturnValue([{ MANAGER_TRACKING: '0', }, vi.fn()]);
+    (await import('@ovh-ux/shell')).initShell = vi.fn().mockResolvedValue(mockedShell('EU'));
+    void renderCookiePolicy();
+    await waitFor(() => {
+      expect(trackingInit).not.toHaveBeenCalled();
+      expect(trackingSetEnabled).toHaveBeenCalled();
```

Comment:
I think we can test that `trackingSetEnabled` is called with `false` parameter:
```suggestion
      expect(trackingSetEnabled).toHaveBeenCalledWith(false);
```